VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/introduction/introduction.component.ts

```diff
@@ -53,7 +53,7 @@ export class IntroductionComponent implements OnInit {
       find(x => x.username.toLocaleLowerCase() === this.adalService.userInfo.userName.toLocaleLowerCase());
     this.videoWebService.raiseParticipantEvent(this.conference.id,
       new UpdateParticipantStatusEventRequest({
-        participant_id: participant.id.toString()
+        participant_id: participant.id.toString(), event_type: EventType.ParticipantJoining
```

Comment:
Same concern as previous issue

Reply:
@shaed-parkar this one is fine. As discussed we can re-use this same method but change the reason to match the status. @valentinabennett has updated the reason.

Reply:
Was discussed with Ru and was resolved